### DISCUSSION TITLE
CocoaPods 快速使用 Swift 静态库 | Alex Lin's Notes

### DISCUSSION THREAD

**Discussion by xwal** [chaoskyme/chaoskyme.github.io#318] [Comment]
https://chaosky.tech/2020/07/28/swift-use-static-framework/ 

 想在项目中使用静态库功能，需要在 Podspec 显示指定 s.static_framework = true，对于多个 Pod 的项目来说，一个个改起来太麻烦了，也不现实。但是 CocoaPods 是 Ruby 写的，我们可以通过 patch CocoaPods 来实现在只写几行代码的情况下，把所有 pod 变成 Static Framework。 通过分析 CocoaPods 的源代码发现，C

**Comment by xiongxiongjiang** [comment]
老哥，你的gittalk挂了。。本来想评论的评论不了。。

**Comment by xwal** [comment]
> @xiongxiongjiang
> 老哥，你的gittalk挂了。。本来想评论的评论不了。。

没挂呢，老哥有啥指教

**Comment by xiongxiongjiang** [comment]
就是评论的时候点github登录会直接跳转到首页，没法登录啊。不是指教哈哈哈，我是觉得你写的这篇文章很好，解决了我的问题。然后我还打算转发到我的博客上并且带上  【转】和你的博客地址，这样可以吧😀

**Comment by xwal** [comment]
> @xiongxiongjiang
> 就是评论的时候点github登录会直接跳转到首页，没法登录啊。不是指教哈哈哈，我是觉得你写的这篇文章很好，解决了我的问题。然后我还打算转发到我的博客上并且带上  【转】和你的博客地址，这样可以吧😀

嗯嗯，昨天github的证书挂了，今天重新生成就可以了。老哥可以转发的，很荣幸能帮助到老哥。
